### ISSUE TITLE
How do i close it?

### ISSUE THREAD

**Issue by ahyde4** [LibreQuest/OculusKiller#72]
The program works perfectly but i just have one little small issue. Im using this with a Quest 2 headset and the only way ive found to close the oculus connection and steam vr is to just unplug the headset entirely. Is there any way to disconnect it other than just doing that?

**Comment by AlephOmicron** [comment]
Did you find out?

**Comment by AlephOmicron** [comment]
You're right, if you close one of the 2 programs your headset will just hang out in a frozen frame, and both oculus and steamvr will actually restart, but not retake control of the hmd until you physically reconnect. it's weird. it's not a huge issue, but if you really want to disconnect without replugging, you should be able to stop the service in task manager. this brings up another issue of steamvr's remote desktop not being able to interact with task manager for some reason.
